### PR TITLE
Initial support for exometer metrics

### PR DESCRIPTION
**Pull request by 0xAX**

### CODE REVIEW

**Comment by GalaxyGorilla** [comment]
I'm currently testing this with the tty and influxdb reporter. Using the ct tests I can see the metrics

- eradius_127.0.0.1:*_server_reset_time_value
- eradius_127.0.0.1:*_server_uptime_value

From the looks of the test I would have expected that metrics like

- client_requests
- client_responses

would show up. Apart from that everything looks good.

**Comment by 0xAX** [comment]
Hello @GalaxyGorilla, the ` test/eradius_logtest.erl ` contains tests for `client_{requests,responses}`, but will add to ct

File: src/eradius_metrics.erl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+subscriptions(MetricsList) ->
+    AvailableMetrics = lists:filter(fun(Metric) -> lists:memeber(Metric, MetricsList) end, ?CLIENT_TYPES),
+    lists:map(fun(MetricName) -> {[eradius, MetricName], value, ?DEFAULT_INTERVAL} end, AvailableMetrics).
+
+update_client_counter_metric(Metric, Value) ->
+    exometer:update_or_create([eradius, Metric], Value, counter, []).
+
+update_nas_prop_metric(Metric, {nas_prop, _, Port, NAS, _, _ ,_, _} = _N, Value) ->
+    Key = binary_to_atom(erlang:iolist_to_binary([NAS, <<":">>, integer_to_binary(Port)]), utf8),
+    exometer:update_or_create([eradius, Key, Metric], Value, counter, []).
```
Comment (surik) [inline src/eradius_metrics.erl]: In context of InfluxDB it would be nice to use NAS host and port as tags but exometer doesn't support tags

**Comment by GalaxyGorilla** [comment]
Hey @0xAX sorry I didn't see that the logtest wasn't part of the ct tests. I executed it as described in the README and I get the metrics which I expected :)

:+1:

**Comment by surik** [comment]
We've tested it with InfluxDB and it works well.

**Pull request merged**